libs/designsystem/src/lib/scss/_global-styles.scss

```
@@ -101,17 +102,19 @@ ion-modal.kirby-overlay {
       }
 
       &:not(.kirby-drawer):not(.kirby-modal-compact) {
+        --height: auto;
+        --min-height: 300px;
+        --max-height: none;
+        --overflow: hidden;
         align-items: flex-start;
         padding-top: size('xl');
-        .modal-wrapper {
-          height: auto;
-          --max-height: none;
-          border-radius: $border-radius;
-
-          &.content-exceeds-viewport {
-            border-bottom-left-radius: 0;
-            border-bottom-right-radius: 0;
-          }
+        transition: all 0.5s ease-in-out, padding 0ms;
```

Consider using transition design tokens from _variables.scss:
```suggestion
        transition: all #{get-transition-duration('long')} #{get-transition-easing('enter-exit')}, padding 0ms;
```